bower.json

```diff
@@ -22,6 +22,6 @@
     "tests"
   ],
   "dependencies": {
-    "ayepromise": "~1.1.2"
+    "es6-promise": "^3.2.1"
```

Comment:
@danielbreves concern when we added ayepromise, was that es6-promise was way bigger than ayepromise, and bigger even than the SDK IIRC.